src/sysl/importers/import_swagger.py

```diff
@@ -213,7 +213,7 @@ def writeEndpoints(self, swag, w):
                         method.upper(),
                         ' ?' if params['query'] else '',
                         '&'.join(
-                            '{}={}{}'.format(p['name'], TYPE_MAP[p['type']], '' if p['required'] else '?')
+                            '{}={}{}'.format(p['name'], 'int' if p['type'] == 'number' else TYPE_MAP[p['type']], '' if p['required'] else '?')
```

Comment:
Not sure I'm reading this right. 'number' isn't always (or even usually) 'int'. If it's correct, it needs a comment.

Reply:
the problem here is the original code is only looking at the `type`. It should also look at `format` to decide sysl type of the query parameter.

I guess I have decided that for query params a `number` is always an integer.

I should remove this and introduce a bit more logic to decide on query param's type.